metamorph/metamorph_plugin.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import logging.config
+import json
+import time
+import os
+
+import requests
+
+
+class MetamorphPlugin(object):
+
+    MINUTE = 60
```

Comment:
I think that something like sleep(60) is perfectly clear to anyone reading to code, and you don't have to maintain an additional variable/constant.

Consider that at some point in the future you determine that you actually want to wait 90 seconds instead of a "minute", now you've got two choice: 1) redefine as self.MINUTE=90 or you're going to  just call sleep(90).